_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Something old, something new…"
+date: 2018-06-20 12:00:00
+categories: preview
+tags: HackIllinois2018, HackIllinois2019, featured
+author: Kyle Begovich
+---
+
+Even though it's only June, planning for HackIllinois 2019 is already well under way, and our Summer Team would love to offer you an inside look at some of our motivations and guiding principles.
+
+# Something Old
+
+The blog has already explored the event last year in a few retrospective pieces, but here we will begin focusing on how it all applies to next year. We are currently in the process of engaging mentors from last year to get their take on their weekend on campus: what they liked and what they would like changed. These mentors are the heart of our hacker experience, tons of fun to host, and exceptionally talented contributors in their field. If you didn't get to engage with the mentors at the event, we hope to make this opportunity more central to the hacker experience this year.
+
+HackIllinois 2018 also saw many engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you may not otherwise have had the opportunity to experience. Our sponsors are some of the biggest, best, and most cutting-edge players in the tech industry, and the value they provide to the event and the attendees cannot be overstated. Being able to take a break from your project is something all of our attendees should take advantage of, and we'll continue to develop these aspects of the event for the years to come.
+
+# Something New
+
+There are only 8 months, 2 days, and 4 hours until check-in, but who's counting? HackIllinois 2019 will be the sixth time Illinois students organize what has become the premier open source event available to undergraduates. While writing this statement, we wanted to explore and emphasize the varied roles that HackIllinois takes on. Our mission this year:
```

Comment:
I like the intro sentence here, but is something else we could do that is time-independent?

Reply:
^^ esp since our date technically is not public yet

Reply:
in the "hackillinois 2019 will be ...." maybe take out "undergraduates" and emphasize the collaboration amongst mentors, hackers, and sponsors through open source (going back to our sponsorship doc idea of emphasizing that unique mentoring experience)

Reply:
I like the time :( @justinyangusa

Reply:
While writing this statement -> While writing our Mission Statement for this year,

Reply:
I left the time in there but rephrased it to be "less than" so it still reads correctly (albeit less relevantly) if you find it much later. For Api's second comment, "available to undergraduates" -> "available to mentors and undergraduates". I'd do more but the rest of the section expands on it quite a bit